src/rpm_ostree/cli_status.rs

```diff
@@ -121,11 +123,42 @@ impl Deployment {
     }
 
     /// return the deployed container image reference
+    /// e.g. ostree-remote-image:fedora:registry:quay.io/fedora/fedora-coreos:stable
     pub fn container_image_reference(&self) -> Option<OstreeImageReference> {
         self.container_image_reference
             .as_ref()
             .and_then(|s| s.as_str().try_into().ok())
     }
+    /// Return the deployed container image as an oci image reference
+    /// but with the digest instead of the tag
+    /// e.g. quay.io/fedora/fedora-coreos@sha256:c4a15145a232d882ccf2ed32d22c06c01a7cf62317eb966a98340ae4bd56dfa6
+    pub fn container_image_reference_digest(&self) -> Option<Reference> {
+        match (
+            &self.container_image_reference(),
+            &self.container_image_reference_digest,
```

Comment:
I found it slightly confusing that this is essentailly a variable name and also the name of this function..

i.e. `self.container_image_reference_digest` and `self.container_image_reference_digest()` are different things (or maybe I'm misunderstanding). 

Might help if we renamed this function `self.get_container_image_reference_digest()` ?

Reply:
good suggestion !